cosmic_ray/modules.py

```diff
@@ -35,6 +36,11 @@ def find_modules(name, excludes=None):
 
     """
     excludes = excludes or []
+    # helps when using bash completion
+    if os.path.exists(name):
+        if name.endswith('.py'):
+            name = name[:-3]
+        name = name.replace('/', '.')
```

Comment:
I'd create a helper function in `cli.py` which contains the code. Just something like:
```
def _fixup_module_name(name):
    if os.path.exists(name):
        if name.endswith('.py'):
            name = name[:-3]
        name = name.replace('/', '.')
    return name
```

Then, in the two places where we use `find_modules()`, use this function to adjust the module name, e.g. `_fixup_module_name(configuration['<top-module>'])`.

Alternatively, create a complete wrapper for `find_modules()` which does this fixup, and then call this wrapper in place of the `find_modules()`. Whichever approach seems cleaner to you.